### DISCUSSION TITLE
chia transaction hash

### DISCUSSION THREAD

**Discussion by AnatoliyPo** [Chia-Network/chia-blockchain#10645] [Support]
I made a transaction from a chia wallet to transfer 2 XCH to another chia wallet. There is a confirmed transfer record in my wallet's transaction log. Where can I see the hash of this transaction?

**Comment by AnatoliyPo** [comment]
Attached is the correspondence. Maybe it will clear things up for those in the know.

Mar 10, 2022, 4:47 GMT+3
We still need you provide the transaction hash.
Could you please contact the support team of your wallet to check it.

Best regards,
SimpleSwap team

	

Stephanie (SimpleSwap)

Mar 9, 2022, 14:43 GMT+3
Let us check this information and get back to you with updates as soon as possible.

Best regards,
SimpleSwap team

	

70samson700

Mar 9, 2022, 14:19 GMT+3

I don't know where I can see the hash of the transaction in the chia wallet. If possible, please tell me where to look. I didn't find it myself. Attached is a screenshot of the chia wallet.

 

Sent from [Mail](https://go.microsoft.com/fwlink/?LinkId=550986) for Windows

 

	

Stephanie (SimpleSwap)

Mar 9, 2022, 11:14 GMT+3
Thank you for your waiting, 
 
Could you please provide the screenshot of your wallet, where the transaction hash, our recipient address and the amount of coins sent are visible.
 
Hope for your understanding and looking forward for your reply.
 
Best regards,
SimpleSwap team
 

	

Stephanie (SimpleSwap)

Mar 9, 2022, 10:59 GMT+3
Thank you,

Your issue will be forwarded to the technical department for further clarification. We will get back to you with updates as soon as possible.

Best regards,
SimpleSwap team
 

	

70samson700

Mar 9, 2022, 10:37 GMT+3

Information as a text:

Transactions

To

Date

Amount

Fee

xch1fpzy5lz92y22fq00p4040wp2e7jth9rkhcu980ewau22kfls6lqqpnxrc0

Confirmed

March 5, 2022 4:01 PM



- 2 XCH

0 XCH

Confirmed at Height

1651654

Memos

Not Available

 

Sent from [Mail](https://go.microsoft.com/fwlink/?LinkId=550986) for Windows

 

	

Stephanie (SimpleSwap)

Mar 9, 2022, 10:16 GMT+3
Thank you,

Could you please provide this information as a text or via link?

Best regards,
SimpleSwap team

	

70samson700

Mar 9, 2022, 10:00 GMT+3

Hello. I don't know where I can see the transaction hash of CHIA. In the wallet CHIA it looks like this:

 

 

Sent from [Mail](https://go.microsoft.com/fwlink/?LinkId=550986) for Windows

 

	

Stephanie (SimpleSwap)

Mar 6, 2022, 9:45 GMT+3
Thank you for your waiting, 

Could you kindly send the transaction hash so that we can sort out your issue?

Thank you!

Best regards,
SimpleSwap team

	

Stephanie (SimpleSwap)

Mar 6, 2022, 9:23 GMT+3
Hi,

Thank you for contacting us.

We will check your swap and get back to you with updates as soon as possible.

Best regards,
SimpleSwap team

	

70samson700

Mar 6, 2022, 8:21 GMT+3

I sent to the wallet indicated in the form 2 . How should I receive confirmation of the transfer?

**Comment by AnatoliyPo** [comment]
I still have the same question: please tell me how can I prove that the transaction to some chia wallet (xch1fpzy5lz92y22fq00p4040wp2e7jth9rkhcu980ewau22kfls6lqqpnxrc0) was made from my wallet. The recipient asks me to give him the hash of the transaction. Where can I see this hash?
In my wallet's transaction log, I only see the amount and the recipient's address. Still available Confirmed at Height
1651654 This Confirmed at Height is the same as Confirmed at Height
in the recipient's wallet.
Sincerely, Anatoly Pogodaev.

**Reply by serge2martel** [reply]
https://xchscan.com/txns/0x5f0d46232e0fcad19d2d2379981e7e62459f8e2c6dd1d58776c3ff97721543f3
